client/lib/abtest/active-tests.js

```diff
@@ -111,10 +111,10 @@ module.exports = {
 	},
 
 	userFirstSignup: {
```

Comment:
I thought about changing it, but realized that since the test results are uniquely identified by test date and we can name our Tracks funnels as we see fit, it just felt like renaming for renaming sake. It felt like "userFirstSignup" was just as accurate for both cases.